### ISSUE TITLE
Bug(?): Automatically limit timestamp backup files

### ISSUE THREAD

**Issue by Alucast** [edkashinsky/reaper-reableton-scripts#4]
At this point it probably feels like I'm a ghost haunting you everyday with my inquiries hahaha 

Hi

I'm having a problem (or maybe user error) with this feature, it doesn't seem to work with me. Yesterday it worked but not today idkw maybe I'm doing something wrong? 

Here's a screencap to visualize it, basically it's not limiting my backups when I save the project. Am I doing this wrong? Is it because I'm using a secondary backup folder? I like to have in in the porject folder but separated in it's own "backup" section, but idk 

https://i.imgur.com/wO5SXxc.mp4

I'd really appreciate knowing what I'm probably doing wrong. Thanks!

**Comment by edkashinsky** [comment]
Hey, @alucast ! Could you specify full title of your project and I will try to repeat this behavior.

By the way, this feature works in background and isn't connected with any action. It based on time ticking. It looks backup time in settings and watch backup files with this delay. For example, you make backups every 1 minute, so the script monitors backups every minute and clear old backups.

But here probably we get some bug

**Comment by Alucast** [comment]
Yeah, so you mean just the project name? It's " Ce Lara - Latinx "  and it's basically the same naming scheme I use with any artist: Artist name dash song name, pretty much.

Ah ok, i thought it worked kind of like amagalmas keep x backup limit which i think he has a manual version and automatic version which I made a custom action to save project and then use the backup limit, but still it would have cleared out the other ones, nice to know! 

thank you for helping me out every time haha anything you need I can help you troubleshoot on my side

Edit: I tried running "Script: amagalma_Backup Limit Stand-Alone - Keep x number of most recent timestamped backups of the selected project(s) (browse to specify).lua" just to see, which makes you look for the main .rpp project file and when selecting it it returns a "No backups needed to be deleted in path : /Users/Alejandro/Desktop/Ce Lara - Latinx/Backups" which is a LIE hahaha so I don't know if these are related or if that information helps you in any way. Curious how both scripts are apparently not working with me. 

I'm using latest Reaper dev version 6.79dev0430 and Mac OS Mojave 10.14.6 in case you wondered.

**Comment by edkashinsky** [comment]
Hey, @alucast! At last I've found time to look closer. I was working on huge refactoring of global startup action and made some fixes for backup observing. Could you synchronise ReaPack and check it out? Don't forget to restart Reaper after synchronization

**Comment by Alucast** [comment]
Hi @edkashinsky! I tried it out and unfortunately I didn't work :( I tried reinstalling the global action and settings scripts and still nothing idkw can I do anything to help pinpoint?

**Comment by edkashinsky** [comment]
I have to close this issue because native support for advanced backup has appeared in Reaper 7, and I have disabled all related features. Everything works out of the box now
